components/sensor/alpha3.rst

```diff
@@ -0,0 +1,89 @@
+Grundfos Alpha3
+===============
+
+.. seo::
+    :description: Instructions for monitoring a Grundfos Alpha3 Model B circulator pump.
+    :image: bedjet.png
+
+The ``alpha3`` component allows you to monitor a Grundfos Alpha3 Model B circulator pump over BLE.
+
+This component supports the following metrics:
+
+- flow rate
+- head pressure
+- power consumption
+- speed (RPM)
+
+This component uses the BLE peripheral on an ESP32, so you also need to enable
+this component. Please see the :doc:`/components/ble_client` docs for how to discover the MAC
+address of your Alpha3 pump, or you can find the list of paired MAC addresses in
+the list of paried Bluetooth devices on your smartphone after using the GO Remote app.
+
+Configuration entry
+-------------------
+
+.. code-block:: yaml
+
+    # Example Configuration
+    esp32_ble_tracker:
+    
+    ble_client:
+      - mac_address: xx:xx:xx:xx:xx:xx
+        id: radiator_pump
+    
+    sensor:
+      - platform: alpha3
+        ble_client_id: radiator_pump
+        flow:
+          name: "Radiator Pump Flow"
+        head:
+          name: "Radiator Pump Head"
+        power:
+          name: "Radiator Pump Power"
+        speed:
+          name: "Radiator Pump Speed"
+    
+
+Configuration variables:
+************************
+
+- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
+- **flow** (*Optional*): The estimated volume flow of the medium in ``m³/h``.
+  - **name** (**Required**, string): The name for the flow sensor.
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **head** (*Optional*): The estimated head (preasure difference generated by the pump) in ``m``.
+  - **name** (**Required**, string): The name for the head sensor.
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **power** (*Optional*): The power consumption of the pump in ``W``.
+  - **name** (**Required**, string): The name for the power sensor.
+  - All other options from :ref:`Sensor <config-sensor>`.
+- **speed** (*Optional*): the speed of the pump rotor in ``RPM``.
+  - **name** (**Required**, string): The name for the speed sensor.
+  - All other options from :ref:`Sensor <config-sensor>`.
+
```

Comment:
There seem to be some missing sensors here that are in the code?

Reply:
I added the sensors to the doc